pgctl/cli.py

```diff
@@ -229,19 +235,27 @@ def __change_state(self, state):
         if state is Start:
             self._run_playground_wide_hook('pre-start')
 
-        run_post_stop_hook = False
         with self.playground_locked():
             failures = self.__locked_change_state(state)
-            if state is Stop:
-                run_post_stop_hook = all(
+
+        # Run the playground wide post-start/post-stop hook
+        # conditioning on the playground is in a fully started/stopped state
+        if state is Start:
+            self._run_playground_wide_hook(
+                'post-start',
+                condition_function=lambda: all(
```

Comment:
this to me is a smell.  we know up front we're not going to call the hook, don't call the function (essentially a fancy boolean trap with 2 levels of indirection instead of just an if statement)